data/styles/clear/include/Icons.mapcss

```diff
@@ -752,6 +752,8 @@ node|z16-[tourism=picnic_site],
 area|z16-[tourism=picnic_site],
 node|z16-[leisure=picnic_table],
 area|z16-[leisure=picnic_table],
+node|z16-[leisure=outdoor_seating],
+area|z16-[leisure=outdoor_seating],
```

Comment:
z16- is too visible for it.

Its ok for picnic tables because they're usually found in outdoor sparsely mapped places.
While outdoor seating is an urban thing. I suggest z18- or z19- even.

Hint: usually visibility and priority of the feature correlate a lot and you put its priority low very rightly.

Reply:
Set to z18-